NitroxServer/Server.cs

```diff
@@ -30,9 +31,10 @@ public Server(ServerConfig config)
             Instance = this;
             worldPersistence = new WorldPersistence();
             world = worldPersistence.Load();
-            packetHandler = new PacketHandler(world);
-            udpServer = new UdpServer(packetHandler, world.PlayerManager, world.EntitySimulation, serverConfiguration);
-            ConsoleCommandProcessor.RegisterCommands();
+            NitroxServiceLocator.InitializeDependencyContainer(new ServerAutoFaqRegistrar(world, serverConfiguration));
```

Comment:
Can you move the AutoFac init to the `Program` class instead? From the AutoFac docs it tells me that's good practice and it makes sense: resolving dependencies is a program-wide issue not a nitrox server issue.

Added benefit is that you can register this `Server` class and let AutoFac handle dependencies instead of doing `NitroxServiceLocator.LocateService<X>()`